### PR TITLE
pop3 capture docs

### PR DESCRIPTION
**Pull request by h00die**
Just a quick little doc for aux server capture pop3.

### CODE REVIEW

File: documentation/modules/auxiliary/server/capture/pop3.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      if pop.mails.empty?
+        puts 'No mail'
+      end
+    end
+    ```
+
+### Output from testing script
+
+When this script is run from teh metasploit console, it intermingles with the commands.
+
```
Comment (ccondon-r7) [inline documentation/modules/auxiliary/server/capture/pop3.md:34]: teh metasploit console > the Metasploit console

File: documentation/modules/auxiliary/server/capture/pop3.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      # check for email, should be none
+      if pop.mails.empty?
+        puts 'No mail'
+      end
+    end
+    ```
+
+### Output from testing script
+
+When this script is run from teh metasploit console, it intermingles with the commands.
```
Comment (ccondon-r7) [inline documentation/modules/auxiliary/server/capture/pop3.md:33]: "teh metasploit console" > the Metasploit console :)

**Comment by smcintyre-r7** [comment]
Merged after a single commit in ca6689cdf4e81af93cb21c68673d1b4407083126 to fix the indentation of the testing script that was causing the triple back quotes to be present in the rendered HTML.

See:

image

So the code block literal should be either indented or enclosed in triple back quotes, but doing both causes the triple back quotes to show up in the rendered HTML.

**Comment by smcintyre-r7** [comment]
# Release Notes
Add module documentation for the pop3 server module that captures credentials.

**Comment by h00die** [comment]
Thanks @smcintyre-r7

**Pull request merged**